go.mod

```diff
@@ -4,25 +4,25 @@ go 1.18
 
 require (
 	github.com/davecgh/go-spew v1.1.1
-	github.com/gcash/bchd v0.18.1
+	github.com/gcash/bchd v0.19.0
 	github.com/gcash/bchlog v0.0.0-20180913005452-b4f036f92fa6
 	github.com/gcash/bchutil v0.0.0-20210113190856-6ea28dff4000
-	github.com/gcash/bchwallet v0.8.3-0.20210524112536-14ca25bc6549
```

Comment:
This is the main weirdness we're fixing.  It has a strange transitive require on a non-existent revision of github.com/gcash/neutrino.  Not all go installs care, but it's not really right.